### PR TITLE
runfix: translation issue for degraded convos [WPB-4370]

### PR DESCRIPTION
**Pull request by tlebon**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-4370" title="WPB-4370" target="_blank">BugWPB-4370</a>  [Web] Incorrect message for new device in German
  </td></table>
  <br />
 

## Description
fix for some broken translations
cleaned a TS error or two also

## Checklist

- [x] PR has been self reviewed by the author;
- [x] Hard-to-understand areas of the code have been commented;
- [x] If it is a core feature, unit tests have been added;

### CODE REVIEW

**Review by thisisamir98 — APPROVED** [review]

File: src/script/components/MessagesList/Message/VerificationMessage.tsx [hunk 1]
```diff
@@ -88,7 +88,9 @@ const VerificationMessage: React.FC<VerificationMessageProps> = ({message}) => {
           <>
             <span className="message-header-plain-sender-name">{nameList}</span>
             <span>
-              {hasMultipleUsers ? t('conversationDeviceStartedUsingMany') : t('conversationDeviceStartedUsingOne')}
+              {hasMultipleUsers || isSelfClient
```
Comment (V-Gira) [inline src/script/components/MessagesList/Message/VerificationMessage.tsx]: sounds like strictly a German hack no?

Comment (tlebon) [reply src/script/components/MessagesList/Message/VerificationMessage.tsx]: Any language with a plural form that's the same as a proper form

Comment (tlebon) [reply src/script/components/MessagesList/Message/VerificationMessage.tsx]: It's possible that an extra translation would be better

Comment (V-Gira) [reply src/script/components/MessagesList/Message/VerificationMessage.tsx]: yeah, there's more than 2 case here, we need you/many/one again I guess.
you formal is 3rd person plural in German, but 3rd person singular in Spanish, 2nd plural in French etc...

Comment (V-Gira) [reply src/script/components/MessagesList/Message/VerificationMessage.tsx]: maybe `conversationDeviceStartedUsing3rdPersonSingular`, `conversationDeviceStartedUsing3rdPersonPlural`, `conversationDeviceStartedUsing2ndPersonSingular`

**Review by V-Gira — APPROVED** [review]

File: src/script/components/MessagesList/Message/VerificationMessage.tsx (hunk 1 above)
Comment (tlebon) [reply src/script/components/MessagesList/Message/VerificationMessage.tsx]: ive used the same format as the other translation location for now. Ill align with Astrid

**Pull request merged**